build.sbt

```diff
@@ -77,7 +77,7 @@ libraryDependencies ++= Seq(
 )
 
 // Compiler settings
-scalacOptions := Seq("-deprecation", "-language:postfixOps", "-opt:l:method", "-Xfuture")
+scalacOptions := Seq("-deprecation", "-language:postfixOps", "-opt:l:method")
```

Comment:
https://github.com/scala/scala/pull/7328

```
[info] Compiling 310 Scala sources and 3 Java sources to /Users/takako.shimamoto/repo/gitbucket/target/scala-2.13/classes ...
[warn] -Xfuture is deprecated: Not used since 2.13.
[warn] one warning found
```

Not sure if it is proper to remove the `-Xfuture`.